index.js

```diff
@@ -70,6 +70,23 @@ server.get('/api/users/:id', (req, res) => {
     .catch(error => res.status(500).json({ error: 'The user information could not be retrieved.' }))
 })
 
+server.delete('/api/users/:id', (req, res) => {
+  const id = Number(req.params.id)
```

Comment:
Does node, behind the scenes, convert this into a number? If not, then this is a great thing to do!